Mathlib/CategoryTheory/Limits/Preserves/Presheaf.lean

```diff
@@ -0,0 +1,179 @@
+/-
+Copyright (c) 2024 Markus Himmel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Markus Himmel
+-/
+import Mathlib.CategoryTheory.Limits.FilteredColimitCommutesFiniteLimit
+import Mathlib.CategoryTheory.Limits.Elements
+
+/-!
+# Finite-limit-preserving presheaves
+
+In this file we prove that if `C` is a small finitely cocomplete category and `A : Cᵒᵖ ⥤ Type u`
+is a presheaf, then `CostructuredArrow yoneda A` is filtered (equivalently, the category of
+elements of `A` is cofiltered) if and only if `A` preserves finite limits.
+
+This is one of the keys steps of establishing the equivalence `Ind C ≌ (Cᵒᵖ ⥤ₗ Type u)` for a
```

Comment:
Here, I feel you should remind the reader that `⥤ₗ` refers to left exact functors.

Reply:
Done.